src/components/tooltip/CallNode.js

```diff
@@ -194,6 +194,16 @@ export class TooltipCallNode extends React.PureComponent<Props> {
       displayData = callTree.getDisplayData(callNodeIndex);
     }
 
+    let libName;
+    if (displayData && displayData.lib) {
+      libName = [
+        <div className="tooltipLabel" key="libName">
+          Library:
+        </div>,
+        displayData.lib,
+      ];
+    }
+
```

Comment:
Thanks for trying this, this is indeed the suggestion in the issue.

But when I tried the code, I found that most of the time this was showing duplicate information with `Resource` or `Script name`. In the case of an add-on, the part of the `script name` was still duplicated.

And actually, the code to get the add-on name is already present below, as this is the code that retrieves the resource (starts with `const resourceIndex = thread.funcTable.resource[funcIndex];`)... but we don't use it when there's a filename.

So here my suggestion:
1. remove your new code here
2. separate `resourceOrFileName` in 2 variables: `fileName` and `resource`. Compute both of them, just move the resource computation out of the `else` block.
3. Insert both of them below.

 And I think this should work just fine for addons.